### PR TITLE
Futurenet

### PR DESCRIPTION
**Pull request by mootz12**
* fix deployment scripts
* update to new blend-sdk

### CODE REVIEW

**Comment by mootz12** [comment]
Note - I can't test this yet against a standalone network. Instead of dealing with things already existing on futurenet, I'm gonna opt to push a "should work" script and do formal testing once new, working standalone containers are deployed.

File: src/external/oracle.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+export interface AssetPrices {
+  price: bigint;
+  assetId: string;
+}
+
+export interface AssetPrices {
+  price: bigint;
+  assetKey: string;
+}
```
Comment (Ryang-21) [inline src/external/oracle.ts:13]: duplicated

Comment (Ryang-21) [reply src/external/oracle.ts:13]: Looks like its not used

**Review by Ryang-21 — COMMENTED** [review]
Had one comment for oracle.ts but besides that lgtm

**Review by Ryang-21 — APPROVED** [review]

File: src/external/oracle.ts (hunk 1 above)
Comment (mootz12) [reply src/external/oracle.ts]: thx. missed this

**Review by Ryang-21 — APPROVED** [review]
lgtm

**Pull request merged**